src/main/resources/aws-smtp-relay.service

```diff
@@ -0,0 +1,14 @@
+# /usr/lib/systemd/system/aws-smtp-relay.service
+[Unit]
+Description=AWS SMTP Relay
+After=network.target
+
+[Service]
+User=aws-smtp-relay
+Type=simple
+ExecStart=/usr/bin/java -jar /usr/share/aws-smtp-relay/aws-smtp-relay-${project.version}-jar-with-dependencies.jar -b 127.0.0.1
```

Comment:
Thanks for all the current changes, i would say the last one would be to make the bining configurable in a /etc/ file with a default of 127.0.0.1